### DISCUSSION TITLE
Custom attributes for user registration Keycloak

### DISCUSSION THREAD

**Discussion by SilvandroOliveira** [keycloak/keycloak#21157] [Q&A]
I need to add custom fields in the registration of a new user from Keycloak, and I saw from the documentation that this is possible if I enable the User Profile Enabled option, but this option is not appearing for me in the master realm with the admin user.

I'm using Keycloak version 21.1.1 and saw the information about this field in the link [here](https://www.keycloak.org/docs/latest/server_admin/#enabling-the-user-profile).

**Comment by danielFesenmeyer** [comment]
Have you started the server with `--features=preview` or `--features=declarative-user-profile`?

It's probably helpful what's displayed on the "master realm - Realm info" page: _\<kc-base-url>/admin/master/console/#/master/info_
Is _DECLARATIVE_USER_PROFILE_ listed as one of the _Enabled features_ or _Disabled features_?
